### PR TITLE
 tweak: Pre-Fare takeover alert layout

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [Pre-Fare Alerts: Update takeover frontend](https://app.asana.com/0/1185117109217413/1204649346992047/f)

Overall, there aren't a ton of changes. Most notable are card bg color, footer, and icons. 

- [ ] Tests added?

### CODE REVIEW

File: assets/css/v2/pre_fare/reconstructed_alert.scss [hunk 1]
```diff
@@ -255,13 +306,14 @@ $alert-card-footer-height: 80px;
...
-  .route-pills__branches__dot {
+    .route-pills__branches
+    .route-pills__branches__dot {
     background-color: #171f26;
   }
 }
 
-.alert-container--takeover, .alert-container--urgent {
+.alert-container--takeover,
+.alert-container--urgent {
```
[the remark this answers is not in the document]
Comment (cmaddox5) [reply assets/css/v2/pre_fare/reconstructed_alert.scss:316]: While takeover does not need it anymore, flexzone alerts still use it. This was a class they shared, so we can just remove a reference to it for takeovers and leave the CSS there for flexzone alerts.

[the remark this answers is not in the document]
Comment (cmaddox5) [reply assets/css/v2/pre_fare/reconstructed_alert.scss:316]: See here: https://github.com/mbta/screens/blob/01c0fff218a0d37d0f019adb1986b130703294ac/assets/src/components/v2/reconstructed_alert.tsx#L35

[the remark this answers is not in the document]
Comment (PaulJKim) [reply assets/css/v2/pre_fare/reconstructed_alert.scss:316]: ahh gotcha

**Comment by PaulJKim** [comment]
Seems odd that backend unit tests are failing when pretty much all the changes are cosmetic 🤔

**Comment by cmaddox5** [comment]
> Seems odd that backend unit tests are failing when pretty much all the changes are cosmetic 🤔

It's because of the branch that this PR is based on. That branch has failing tests because I put the tests in their own branch. So until the test PR is merged, CI will fail here and in #1785.

**Review by PaulJKim — APPROVED** [review]
LGTM!

**Pull request merged**